pkg/virt-launcher/virtwrap/BUILD.bazel

```diff
@@ -1,4 +1,5 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load("@kubevirt//tools/ginkgo:ginkgo.bzl", "ginkgo_test")
```

Comment:
Can you please explain this line?

Reply:
Just the target import as I need it to be able to call `ginkgo_test`.
The workaround for running parallel ginkgo with bazel was introduced in https://github.com/kubevirt/kubevirt/pull/5749.